spec/controllers/admin/users_controller_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe Admin::UsersController, type: :controller do
```

Comment:
I think I want to vote no on adding controller specs. I think they may officially be deprecated upstream now, anyway. Request specs look similar but provide better coverage, or system specs provide a more user-oriented perspective for similar code, and much more coverage as well.

I've dabbled with unit-testing controllers but it's a little weird and mock-heavy and the rails controller-type specs aren't a good way to achieve that IMO.

Reply:
I'm usually a request spec person as well, I was just keeping with existing convention for the scope of this ticket.

Reply:
aha, i was out of context. carry on for now, then

Reply:
I'm okay if we want to move everything over to request specs. I don't have strong feelings one way or the other.